server/events.go

```diff
@@ -721,37 +721,37 @@ func (s *Server) initEventTracking() {
 	}
 	monSrvc := map[string]msgHandler{
 		"STATSZ": s.statszReq,
-		"VARZ": func(sub *subscription, _ *client, subject, reply string, msg []byte) {
+		"VARZ": func(sub *subscription, c *client, subject, reply string, msg []byte) {
```

Comment:
These are changes that were done in https://github.com/nats-io/nats-server/pull/2338. Should should have rebased instead so that this PR shows only your changes? (it's ok if you don't)